dependencies.props

```diff
@@ -1,14 +1,71 @@
+    <CoreClrCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</CoreClrCurrentRef>
+    <StandardCurrentRef>824797963a7c7cfce78899f9c1af01e1011a1bd9</StandardCurrentRef>
   </PropertyGroup>
-
-  <!-- Create a collection of all project.json files for dependency updates. -->
+  
+  <!-- Auto-upgraded properties for each build info dependency. -->
+  <PropertyGroup>
+	<CoreFxExpectedPrerelease>preview2-25301-02</CoreFxExpectedPrerelease>
+	<CoreClrExpectedPrerelease>preview2-25301-02</CoreClrExpectedPrerelease>
+    <StandardExpectedPrerelease>preview2-25301-01</StandardExpectedPrerelease>
+  </PropertyGroup>
+  
+  <!-- Package dependency verification/auto-upgrade configuration. -->
+   <PropertyGroup>
+      <BaseDotNetBuildInfo>build-info/dotnet/</BaseDotNetBuildInfo>
+      <DependencyBranch>master</DependencyBranch>
+      <CurrentRefXmlPath>$(MSBuildThisFileFullPath)</CurrentRefXmlPath>
+  </PropertyGroup>
+  
   <ItemGroup>
-    <_ProjectJsonExcludes Include="$(SourceDir)test\**\project.json" />
-    <_ProjectJsonExcludes Include="$(SourceDir)pkg\**\project.json" />
-    <ProjectJsonFiles Include="$(SourceDir)**\project.json" Exclude="@(_ProjectJsonExcludes)" />
-  </ItemGroup>
+	<RemoteDependencyBuildInfo Include="CoreFx">
+		<BuildInfoPath>$(BaseDotNetBuildInfo)corefx/$(DependencyBranch)</BuildInfoPath>
+		<CurrentRef>$(CoreFxCurrentRef)</CurrentRef>
+	</RemoteDependencyBuildInfo>
+	<RemoteDependencyBuildInfo Include="CoreClr">
+		<BuildInfoPath>$(BaseDotNetBuildInfo)coreclr/$(DependencyBranch)</BuildInfoPath>
+		<CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
+	</RemoteDependencyBuildInfo>
+	<RemoteDependencyBuildInfo Include="Standard">
+		<BuildInfoPath>$(BaseDotNetBuildInfo)standard/$(DependencyBranch)</BuildInfoPath>
+		<CurrentRef>$(CoreClrCurrentRef)</CurrentRef>
+	</RemoteDependencyBuildInfo>
+
+	<DependencyBuildInfo Include="@(RemoteDependencyBuildInfo)">
+		<RawVersionsBaseUrl>https://raw.githubusercontent.com/dotnet/versions</RawVersionsBaseUrl>
+	</DependencyBuildInfo>
+	
+	<XmlUpdateStep Include="CoreFx">
+		<Path>$(MSBuildThisFileFullPath)</Path>
+		<ElementName>CoreFxExpectedPrerelease</ElementName>
+		<BuildInfoName>CoreFx</BuildInfoName>
+	</XmlUpdateStep>
+	<XmlUpdateStep Include="CoreClr">
+		<Path>$(MSBuildThisFileFullPath)</Path>
+		<ElementName>CoreClrExpectedPrerelease</ElementName>
+		<BuildInfoName>CoreClr</BuildInfoName>
+	</XmlUpdateStep>
+    <XmlUpdateStep Include="Standard">
+		<Path>$(MSBuildThisFileFullPath)</Path>
+		<ElementName>StandardExpectedPrerelease</ElementName>
+		<BuildInfoName>Standard</BuildInfoName>
+	</XmlUpdateStep>
+</ItemGroup>
+  
+  <!-- Copied Microsoft.NETCore.Runtime.CoreCLR and Microsoft.NETCore.Runtime.JIT versions -->
+  <PropertyGroup>
+       <CoreCLRVersion>2.0.0-$(CoreClrExpectedPrerelease)</CoreCLRVersion>
+       <JitVersion>2.0.0-$(CoreClrExpectedPrerelease)</JitVersion>
+  </PropertyGroup>
 </Project>
```

Comment:
It would be slightly better for Build from Source if you moved the ProjectJsonFiles items into this file, like in CoreFX [dependencies.props#L177](https://github.com/dotnet/corefx/blob/dde2d618cc02fe1f226c7448b6bc9e309f0aede4/dependencies.props#L177).

Right now the workaround is to specify them in core-setup.props (like they are now in [corefx.props](https://github.com/dotnet/source-build/blob/3111bcb8c8307f29a7f9ffa4e29cc39647e4bd43/targets/corefx.props#L15), although that can be removed now).